corehq/apps/app_manager/urls.py

```diff
@@ -128,6 +129,7 @@
     url(r'^download/(?P<app_id>[\w-]+)/$', 'download_index', name='download_index'),
     url(r'^download/(?P<app_id>[\w-]+)/CommCare.ccz$', DownloadCCZ.as_view(),
         name=DownloadCCZ.name),
+    url(r'^download/(?P<app_id>[\w-]+)/multimedia/', include(media_download_urls)),
```

Comment:
After fixing the multimedia zip bug on master the download URL always 404'd because it expected the zip to be attached to the saved app. Bumping this include (which is just a single URL) up in the order fixes it.